services/restic_backup.md

```diff
@@ -115,6 +115,34 @@ For mariadb do the following:
 
 ## Restore from a backup
 
+To restore from a GUI 
+
+  1. Install the restic-browser app with 
+
+    ```bash
+    brew install restic-browser
+    ```
+
+  2. Launch the application
+
+     Create a new Location:
+
+       * In the **Type**: Select Google Cloud Storage
+       * In the **Bucket**: Enter the name and path of your bucket. (e.g., gs:postgres-15-backup:daily)
+       * In the **GOOGLE_PROJECT_ID**: Enter pul-gcdc
+       * In the **GOOGLE_APPLICATION_CREDENTIALS**: Get a copy of the credentials and place them at a know location
+         * Log into `lib-postgres-prod1` and download or copy the credentials with the following
+           ```bash
+           sudo su - postgres
+           cat ~/.restic/pul-gcdc-UUIDnumber.json
+           ```
+       * In the **Repository Password**: Get the contents from the server
+         * Log into `lib-postgres-prod1` and get the password with the following
+           ```bash
+           sudo su - postgres
+           cat ~/.restic.pwd
+           ```
+
```

Comment:
These instructions worked for me. I was able to configure a location and then browse the list of files in the restic browser. It was pretty slow to load them, I thought at first something must still be configured wrong but eventually it loaded a long list of files.